cmd/skoap/main.go

```diff
@@ -0,0 +1,318 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/zalando-incubator/skoap"
+	"github.com/zalando/skipper"
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/filters/builtin"
+	"github.com/zalando/skipper/proxy"
+	"github.com/zalando/skipper/routing"
+)
+
+const (
+	addressFlag        = "address"
+	defaultEtcdPrefix  = "/skipper"
```

Comment:
Should skoap have a different default prefix?
Maybe you want to run skipper and skoap with one etcd?